CONTRIBUTING.md

```diff
@@ -83,13 +87,19 @@ Now, all dependencies are installed into the Python environment you chose, which
 pdm run test
 ```
 
+faster test using pytest-xdist
+
+```bash
+pdm run test -n local
```

Comment:
`local` is an invalid value, do you mean `auto`?

Reply:
yes...